src/scripts.js

```diff
@@ -55,13 +52,13 @@ Promise.all([fetchedUserData, fetchedRecipeData, fetchedIngredientData])
     generateUser(values[0])
     createRecipeDataset(values[1])
     addRecipeNameAndCost(values[2])
-    addPantryIngredientNames(values[2])
+    // currentUser.addPantryIngredientNames(values[2])
     loadWebsite()
   })
 
 function loadWebsite() {
   createRecipeCards()
-  displayPantryInfo(currentUser)
+  domUpdates.addPantryInfoToDom(currentUser.alphabetizePantry())
```

Comment:
I'm not totally sure about this syntax. I would be open to suggestions about readability or something

Reply:
I went ahead and assigned that user method to a variable then passed it in for readability

Reply:
looks ok to me... pretty sure we're doing a similar thing in other places